<?xml version="1.0"?>

<launch>

    <node pkg="gmp_test" type="orient_gmp_test" name="orient_gmp_test_node"
          output="screen" cwd="node">

        <rosparam>

          train_filename: "src/simulation/data/orient_data.bin"
          results_filename: "src/simulation/data/orient_gmp_test_results.bin"

          train_method: "LS"
          N_kernels: 25
          kernels_std_scaling: 1.5

          scale_type: "rot_wb" # {"prop", "rot_min", "rot_wb"}
          wb_normal: [0, 0, 1] # work-bench normal

          Qg: [0.7361, 0.3551, 0.4142, -0.4007] # new target for execution
          T: 8.33 # new time duration for execution

          orient_sim_fun: "Cart" # {"log", "Cart"}

          read_gmp_from_file: false
          write_gmp_to_file: false
          gmp_filename: "src/simulation/data/gmp_orient.bin"

        </rosparam>

    </node>

</launch>
